src/tests/unit/components/cards-list/CardsList.spec.jsx

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from '@testing-library/react'
+import { describe, expect, vi } from 'vitest'
+import userEvent from '@testing-library/user-event'
+import CardsList from '~/components/cards-list/CardsList'
+
+describe('Card List', () => {
+  let loading = true
+  let cards = []
+  test('renders Loader when loading and no cards', () => {
+    render(<CardsList cards={cards} loading={loading} />)
+    const loader = screen.getAllByTestId('loader')[0]
+    expect(loader).toBeInTheDocument()
+  })
+  test('shows card when cards length is', () => {
+    let cards = ['test']
+    render(<CardsList cards={cards} loading={loading} />)
+    const card = screen.getByText('test')
+    expect(card).toBeInTheDocument()
```

Comment:
is this the correct behavior given that loading is true?

Reply:
yes